javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
In our case we already lose that before sending to the extension, but could be useful to support. Do you see a easy way for that, except checking instances of all the sub-classes?